runtime/tests/runtime_integration_tests/common.rs

```diff
@@ -241,11 +246,14 @@ pub fn start_rt_test_pqc_model(
         },
         ..Default::default()
     });
-
-    let image =
-        caliptra_builder::build_and_sign_image(fmc_fwid, runtime_fwid, image_options).unwrap();
-
-    let (vendor_pk_hash, owner_pk_hash) = image_pk_desc_hash(&image.manifest);
+    init_params.fuses = Fuses {
+        fuse_pqc_key_type: pqc_key_type as u32,
+        vendor_pk_hash,
+        owner_pk_hash,
+        soc_manifest_svn: svn_to_bitmap(args.soc_manifest_svn.unwrap_or(0)),
+        soc_manifest_max_svn: args.soc_manifest_max_svn.unwrap_or(127) as u8,
+        ..Default::default()
+    };
```

Comment:
This looks like it would always override fuses that are passed in, right?

Should this instead happen in the init block on line 237?

Reply:
Okay sounds good, I guess the tests that care about this re-implement some of this common code anyway.